pkg/sql/sem/tree/backup.go

```diff
@@ -145,6 +145,7 @@ type RestoreOptions struct {
 	SchemaOnly                       bool
 	VerifyData                       bool
 	UnsafeRestoreIncompatibleVersion bool
+	ExecutionLocality                Expr
```

Comment:
Since this is an Expr, is there a Walk function we need to update? I'm never sure about that.